AppEngine Java8 Nightly failures...

```
------------------------------------------------------------
- testing appengine-java8/endpoints-v2-backend
------------------------------------------------------------
[ERROR] Project does not define required minimum version of Maven.
[ERROR] Update the pom.xml to contain maven-enforcer-plugin to
[ERROR] force the Maven version which is needed to build this project.
[ERROR] See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[ERROR] Using the minimum version of Maven: 3.0.5

 Testing completed.

------------------------------------------------------------
- testing appengine-java8/endpoints-v2-guice
------------------------------------------------------------
[ERROR] Project does not define required minimum version of Maven.
[ERROR] Update the pom.xml to contain maven-enforcer-plugin to
[ERROR] force the Maven version which is needed to build this project.
[ERROR] See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[ERROR] Using the minimum version of Maven: 3.0.5

 Testing completed.

------------------------------------------------------------
- testing appengine-java8/endpoints-v2-migration
------------------------------------------------------------
[ERROR] Project requires an incorrect minimum version of Maven.
[ERROR] Update the pom.xml to contain maven-enforcer-plugin to
[ERROR] force the Maven version which is needed to build this project.
[ERROR] See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[ERROR] Using the minimum version of Maven: 3.0

 Testing completed.

------------------------------------------------------------
- testing appengine-java8/springboot-helloworld
------------------------------------------------------------
[ERROR] Project does not define required minimum version of Maven.
[ERROR] Update the pom.xml to contain maven-enforcer-plugin to
[ERROR] force the Maven version which is needed to build this project.
[ERROR] See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[ERROR] Using the minimum version of Maven: 3.0.5

 Testing completed.
```

--- a/appengine-java8/bigtable/pom.xml
+++ b/appengine-java8/bigtable/pom.xml
@@ -90,20 +90,6 @@ limitations under the License.
     <!-- for hot reload of the web application-->
     <outputDirectory>${project.build.directory}/${project.build.finalName}/WEB-INF/classes</outputDirectory>
     <plugins>
-      <!--<plugin>-->
-        <!--<groupId>org.codehaus.mojo</groupId>-->
-        <!--<artifactId>versions-maven-plugin</artifactId>-->
-        <!--<version>2.4</version>-->
-        <!--<executions>-->
-          <!--<execution>-->
-            <!--<phase>compile</phase>-->
-            <!--<goals>-->
-              <!--<goal>display-dependency-updates</goal>-->
-              <!--<goal>display-plugin-updates</goal>-->
-            <!--</goals>-->
-          <!--</execution>-->
-        <!--</executions>-->
-      <!--</plugin>-->
 
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>

--- a/appengine-java8/endpoints-v2-backend/pom.xml
+++ b/appengine-java8/endpoints-v2-backend/pom.xml
@@ -116,20 +116,6 @@
         </configuration>
       </plugin>
       <!-- [END endpoints_plugin] -->
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>versions-maven-plugin</artifactId>
-        <version>2.7</version>
-        <executions>
-          <execution>
-            <phase>compile</phase>
-            <goals>
-              <goal>display-dependency-updates</goal>
-              <goal>display-plugin-updates</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 </project>

--- a/appengine-java8/endpoints-v2-guice/pom.xml
+++ b/appengine-java8/endpoints-v2-guice/pom.xml
@@ -118,20 +118,6 @@
           </serviceClasses>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>versions-maven-plugin</artifactId>
-        <version>2.7</version>
-        <executions>
-          <execution>
-            <phase>compile</phase>
-            <goals>
-              <goal>display-dependency-updates</goal>
-              <goal>display-plugin-updates</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 </project>

--- a/appengine-java8/endpoints-v2-migration/pom.xml
+++ b/appengine-java8/endpoints-v2-migration/pom.xml
@@ -87,20 +87,6 @@ limitations under the License.
           <hostname>YOUR-PROJECT-ID.appspot.com</hostname>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>versions-maven-plugin</artifactId>
-        <version>2.7</version>
-        <executions>
-          <execution>
-            <phase>compile</phase>
-            <goals>
-              <goal>display-dependency-updates</goal>
-              <goal>display-plugin-updates</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
     </plugins>
   </build>
 

--- a/appengine-java8/springboot-helloworld/pom.xml
+++ b/appengine-java8/springboot-helloworld/pom.xml
@@ -94,27 +94,6 @@
         </configuration>
       </plugin>
 
-       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>versions-maven-plugin</artifactId>
-        <version>2.7</version>
-        <executions>
-          <execution>
-            <phase>compile</phase>
-            <goals>
-              <goal>display-dependency-updates</goal>
-              <goal>display-plugin-updates</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <excludes>
-            <exclude>javax.servlet:javax.servlet-api</exclude>
-            <exclude>com.google.guava:guava</exclude> <!-- avoid android version -->
-          </excludes>
-        </configuration>
-      </plugin>
-
       <plugin>
         <artifactId>maven-war-plugin</artifactId>
         <version>3.3.1</version>

